src/format/stream.rs

```diff
@@ -24,6 +27,14 @@ extern "C" {
         value: *const c_char,
     ) -> c_int;
     fn ffw_stream_set_id(stream: *mut c_void, id: c_int);
+    fn ffw_stream_get_nb_side_data(stream: *const c_void) -> c_uint;
+    fn ffw_stream_get_side_data(stream: *const c_void, index: c_uint) -> *mut c_void;
+    fn ffw_stream_add_side_data(
+        stream: *mut c_void,
+        data_type: c_int,
+        data: *const c_void,
+        size: u64,
+    ) -> c_int;
```

Comment:
Be careful with types. C type `size_t` isn't always a 64-bit unsigned integer.
```suggestion
    fn ffw_stream_add_side_data(
        stream: *mut c_void,
        data_type: c_int,
        data: *const u8,
        size: usize,
    ) -> c_int;
```

Reply:
Done.